x/market/internal/types/params.go

```diff
@@ -44,21 +42,19 @@ var _ subspace.ParamSet = &Params{}
 
 // Params market parameters
 type Params struct {
-	PoolRecoveryPeriod   int64        `json:"pool_recovery_period" yaml:"pool_recovery_period"`
-	BasePool             sdk.Dec      `json:"base_pool" yaml:"base_pool"`
-	MinSpread            sdk.Dec      `json:"min_spread" yaml:"min_spread"`
-	TobinTax             sdk.Dec      `json:"tobin_tax" yaml:"tobin_tax"`
-	IlliquidTobinTaxList TobinTaxList `json:"illiquid_tobin_tax_list" yaml:"illiquid_tobin_tax_list"`
+	PoolRecoveryPeriod int64   `json:"pool_recovery_period" yaml:"pool_recovery_period"`
+	BasePool           sdk.Dec `json:"base_pool" yaml:"base_pool"`
+	MinSpread          sdk.Dec `json:"min_spread" yaml:"min_spread"`
+	TobinTax           sdk.Dec `json:"tobin_tax" yaml:"tobin_tax"`
 }
 
 // DefaultParams creates default market module parameters
 func DefaultParams() Params {
 	return Params{
-		BasePool:             DefaultBasePool,
-		PoolRecoveryPeriod:   DefaultPoolRecoveryPeriod,
-		MinSpread:            DefaultMinSpread,
-		TobinTax:             DefaultTobinTax,
-		IlliquidTobinTaxList: DefaultIlliquidTobinTaxList,
+		BasePool:           DefaultBasePool,
+		PoolRecoveryPeriod: DefaultPoolRecoveryPeriod,
+		MinSpread:          DefaultMinSpread,
```

Comment:
What do you think about renaming `MinSpread` to `MinStabilitySpread`? Given that the `MinSpread` only applies to swaps involving Luna, think we need to differentiate this variable name somehow to remind people of that restriction

Reply:
agree,

Reply:
Awesome